File: Famoser.SyncApi/Entities/Api/DeviceEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Famoser.SyncApi.Entities.Api.Base;

namespace Famoser.SyncApi.Entities.Api
{
    public class DeviceEntity : BaseEntity
    {
        public Guid UserId { get; set; }
        public string Name { get; set; }
        public DateTime CreateDateTime { get; set; }
        public DateTime LastRequestDateTime { get; set; }
    }
}

File: Famoser.SyncApi.Webpage/src/Models/Communication/Entities/DeviceEntity.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 04.11.2016
 * Time: 19:06
 */

namespace Famoser\SyncApi\Models\Communication\Entities;


use Famoser\SyncApi\Models\Communication\Entities\Base\BaseEntity;

class DeviceEntity extends BaseEntity
{
    /* @var string $UserId type_of:guid */
    public $UserId;
}